traitsui/tests/editors/test_range_editor_text.py

```diff
@@ -68,39 +70,20 @@ def test_wx_text_editing(self):
         # (tests a bug where this fails with an AttributeError)
 
         num = NumberWithRangeEditor()
-        with reraise_exceptions(), create_ui(num) as ui:
-
+        tester = UITester()
+        with tester.create_ui(num) as ui:
             # the following is equivalent to setting the text in the text
             # control, then pressing OK
-
-            textctrl = ui.control.FindWindowByName("text")
-            textctrl.SetValue("1")
+            text = tester.find_by_name(ui, "number").locate(locator.WidgetType.textbox)
+            text.perform(command.KeyClick("1"))
+            text.perform(command.KeyClick("Enter"))
 
         # the number traits should be between 3 and 8
         self.assertTrue(3 <= num.number <= 8)
 
-    @requires_toolkit([ToolkitName.qt])
-    def test_avoid_slider_feedback(self):
```

Comment:
For future reference, I believe this is removed because the same scenario is now tested in test_range_editor.py